### DISCUSSION TITLE
Is there a design or elements that can be farmed/delegated out ?

### DISCUSSION THREAD

**Discussion by HexBlit** [telosnetwork/open-block-explorer#17] [Q&A]
I have variable time and want to help push this forward. 

Happy to make charts look nicer, (I have a lot of experience in this) or build components that can be brought into the pages. or if there are specifically something that is needed that is not started, hoping to follow a design/vision. reducing refactor to get it designed the way that screams Telos is awesome.

**Comment by donnyquixotic** [comment]
Hi @FullDevMike, generally speaking, _any_ unassigned open issues are available for public contribution via PR.  

That said, I have some small design update tasks to get you started:

General Style Updates: 
- latest transactions table should reflect the  [design prototype](https://www.figma.com/proto/wy5BIdz4Gx3UiI3FFpuegk/Open-Explorer?node-id=41%3A1946&starting-point-node-id=12%3A80) Add/Update font style, containers & background gradients (e.g. data column)
- search bar drop down: seed the "results" with a mock array of strings for now and style according to [design prototype](https://www.figma.com/proto/wy5BIdz4Gx3UiI3FFpuegk/Open-Explorer?node-id=41%3A1946&starting-point-node-id=12%3A80)
- 24hr price chart & volume stats: update styling to match design (axis, labels, the data point popup styling, etc). for this one we don't want to do anything anti-pattern or too against the grain for highcharts so only do it if it easily allows it or low effort to add.  

There are plenty of tasks of varying effort available if/when you finish these and would like more. Let me know if you have any questions. Feel free to post here or send me a message on Telegram. For future tasks I'll have more explicit acceptance criteria, if you need any clarification do not hesitate to ask. Happy to have you contributing!

Note: utilize [quasar components](https://quasar.dev/vue-components) when appropriate & available before making a custom component
